### PR TITLE
Update Readme

### LINKED ISSUE #255: Update README

### PR DESCRIPTION
**Pull request by Incogdino**
Closes #255

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `konTActs` because it is developed for CS2030S TAs to manage their students, store their github details and other contacts like professors and fellow TAs. 
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://ay2425s1-cs2103t-t11-2.github.io/tp/)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
 
+**The project simulates an ongoing software project for a desktop application (called _konTActs_) used for managing contact details.
+  * For details on how to use konTActs, please refer to the user guide for [konTActs](https://ay2425s1-cs2103t-t11-2.github.io/tp/UserGuide.html)
```
Comment (SwaminathanViswa) [inline README.md:17]: Better to standardise to `KonTActs` since line 12 & 21 it is written as such.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
   * as a tool to help them keep track of their students.
-* The project simulates an ongoing software project for a desktop application (called _konTActs_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `konTActs` because it is developed for CS2030S TAs to manage their students, store their github details and other contacts like professors and fellow TAs. 
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://ay2425s1-cs2103t-t11-2.github.io/tp/)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
 
+**The project simulates an ongoing software project for a desktop application (called _konTActs_) used for managing contact details.
```
Comment (SwaminathanViswa) [inline README.md:16]: Same comment as the comment for line 17.

**Review by SwaminathanViswa — COMMENTED** [review]
LGTM! just a bit or grammar and naming convention fixes!

File: README.md (hunk 2 above)
Comment (Incogdino) [reply README.md:16]: Okay gotcha

**Review by SwaminathanViswa — APPROVED** [review]
LGTM!

**Pull request merged**